engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/openapi/StandaloneOpenApiGenerator.scala

```diff
@@ -0,0 +1,76 @@
+package pl.touk.nussknacker.engine.standalone.openapi
+
+import io.circe.Json
+import io.circe.syntax._
+import io.circe.generic.auto._
+import pl.touk.nussknacker.engine.standalone.StandaloneProcessInterpreter
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+object StandaloneOpenApiGenerator {
+
+  private val OPEN_API_VERSION = "3.0.0"
+
+  private val jsonEncoder = BestEffortJsonEncoder(failOnUnkown = true, getClass.getClassLoader)
+
+  private[standalone] def generateScenarioDefinition(processName: String,
+                                                     requestDefinition: Json,
+                                                     responseDefinition: Json,
+                                                     description: Map[String, String],
+                                                     tags: List[String]
+                                                    ): Json = {
+   val postOpenApiDefinition = generatePostOApiDefinition(
```

Comment:
Why do we handle only POST?
If we don't want to implement it, we should at least return None for GET etc.?

Reply:
You are right, I have added a slot to put GET definition in the future but marked it with TODO for now.